airflow_dbt_python/__version__.py

```diff
@@ -2,4 +2,4 @@
 __author__ = "Tomás Farías Santana"
 __copyright__ = "Copyright 2021 Tomás Farías Santana"
 __title__ = "airflow-dbt-python"
-__version__ = "1.0.5"
+__version__ = "1.1.0"
```

Comment:
Due to the deprecations, and general instability of supporting multiple dbt versions, I think we should go with a v2.0. I can push a new PR to do that before release. Ultimately, we need a better translation layer with dbt: I've been toying with a `dbt-lib` for a while, but the incentive is currently not there. Anyways...